docs/src/man/hosting.md

```diff
@@ -340,6 +339,25 @@ or `git@`.
 See the [`deploydocs`](@ref) function documentation for more details.
 
 
+## Using Simple SSH
+
+If you do not have access to either Travis or GitHub for automated documentation updates,
+or you are working in a private Git hosting service where using these is impractical, it
+may be desirable to push objects more simply via SSH.  For such cases, simply call, e.g.
+
+```julia
+"deploy" ∈ ARGS && deploydocs(
+    repo = "github.com/USER_NAME/PACKAGE_NAME.jl.git",
+    deploy_config = Documenter.SimpleSSHConfig("/path/to/ssh-key")
+)
+```
+
+Note that calling `deploydocs` with the `SimpleSSHConfig` will always deploy, so if you
+are worried about pushing to the remote Git repository you may want to guard against it
+by, as in this example, checking if the argument `"deploy"` was passed on the command line.
```

Comment:
Small rephrasing suggestion:

```suggestion
Note that calling `deploydocs` with the `SimpleSSHConfig` will always deploy. So, to avoid
unwanted accidental deployments, it is good practice to guard the `deplodocs` call, e.g. by
checking whether `"deploy"` was passed as a command line argument to the `make.jl` script
(as is done in the example above).
```